src/TestFramework/Version/VersionParser.php

```diff
@@ -46,16 +45,20 @@ final class VersionParser
 {
     private const VERSION_REGEX = '/(?<version>\d+\.\d+\.?\d*)(?<prerelease>-[0-9a-zA-Z.]+)?(?<build>\+[0-9a-zA-Z.]+)?/';
 
+    /**
+     * @throws InvalidVersion
+     */
     public function parse(string $content): string
     {
         $matches = [];
         $matched = preg_match(self::VERSION_REGEX, $content, $matches);
 
-        Assert::notSame(
-            $matched,
-            0,
-            sprintf('Expected "%s" to be contain a valid SemVer (sub)string value.', $content)
-        );
+        if ($matched === 0) {
```

Comment:
`$matched !== 1` ? since negative return values are `0`, `false`

Reply:
since it's the safe function it's going to be `0` or `1` and throw an error on `false`